python/tests/voice/test_voice_spans_gemini.py

```
@@ -0,0 +1,564 @@
+    # Text-only voiced message → _extract_audio_from_message returns None, so the
+    # barge-in skips send_audio and isolates the native interrupt() span.
+    await ScenarioExecutor._fire_user_interrupt(
+        executor, {"role": "user", "content": "wait, stop"}
+    )
+
+    if not pending.done():
+        pending.cancel()
+        try:
+            await pending
+        except (asyncio.CancelledError, Exception):
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)